Handle/Eula.txt

```diff
@@ -0,0 +1,75 @@
+﻿Sysinternals Software License Terms
```

Comment:
Please never check in binaries in git. 
It stays in the git history forever, and bloats the size of the repository, significantly. 
Anything that's an exe can either be download from somewhere or accessed on a share.